app/content/tests/test_user_model.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from app.content.factories import RegistrationFactory, UserFactory
+from app.forms.enums import EventFormType
+from app.forms.tests.form_factories import EventFormFactory, SubmissionFactory
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture()
+def user():
+    return UserFactory()
+
+
+def test_get_unanswered_evaluations_when_has_no_unanswered_evaluations_returns_no_evaluations(
+    user,
+):
+    """
+    Test that no unanswered evaluations are returned
+    when the user has attended an event and answered the evaluation.
+    """
+    evaluation = EventFormFactory(type=EventFormType.EVALUATION)
+    RegistrationFactory(event=evaluation.event, user=user, has_attended=True)
+    SubmissionFactory(form=evaluation, user=user)
+
+    unanswered_evaluations = user.get_unanswered_evaluations()
+
+    assert not unanswered_evaluations.exists()
+
+
+def test_get_unanswered_evaluations_when_has_unanswered_evaluations_returns_unanswered_evaluations(
+    user,
+):
+    """
+    Test that all unanswered evaluations are returned
+    when the user has attended an event and not answered the evaluation.
+    """
+    evaluation = EventFormFactory(type=EventFormType.EVALUATION)
+    RegistrationFactory(event=evaluation.event, user=user, has_attended=True)
+
+    unanswered_evaluations = user.get_unanswered_evaluations()
+
+    assert unanswered_evaluations.count() == 1
+
+
+def test_has_unanswered_evaluations_when_has_unanswered_evaluations_returns_false(user):
```

Comment:
Navnet på denne og testen under må vel byttes om?
```suggestion
def test_has_unanswered_evaluations_when_has_no_unanswered_evaluations_returns_false(user):
```